pkgs/by-name/bi/bitbox-bridge/package.nix

```
@@ -34,6 +34,13 @@ rustPlatform.buildRustPackage (finalAttrs: {
     libudev-zero
   ];
 
+  postInstall = lib.optionalString stdenv.hostPlatform.isLinux ''
+    mkdir -p $out/lib/systemd/user
+    substitute ${src}/bitbox-bridge/release/linux/bitbox-bridge.service $out/lib/systemd/user/bitbox-bridge.service \
+      --replace-fail /opt/bitbox-bridge/bin/bitbox-bridge $out/bin/bitbox-bridge
+    install -Dm644 ${src}/bitbox-bridge/release/linux/hid-digitalbitbox.rules $out/lib/udev/rules.d/50-hid-digitalbitbox.rules
```

Why not just this instead?

```suggestion
    substitute bitbox-bridge/release/linux/bitbox-bridge.service $out/lib/systemd/user/bitbox-bridge.service \
      --replace-fail /opt/bitbox-bridge/bin/bitbox-bridge $out/bin/bitbox-bridge
    install -Dm644 bitbox-bridge/release/linux/hid-digitalbitbox.rules $out/lib/udev/rules.d/50-hid-digitalbitbox.rules
```

---

Ahh ok I'll try this then we might be able to remove the rec as well